### DISCUSSION TITLE
[Weekly Curation Report]: November 03 - 13

### DISCUSSION THREAD

**Discussion by AlexanderPico** [wikipathways/wikipathways-help#84] [curation-reports]
### Recent changes

 * 27 edits reviewed
 * Mostly fixes to literature references, node types, descriptions
 
_Thanks to @larsgw  and @khanspers for the bulk of the work to fix references this week!_
  

### Under construction

 * 3 actively under construction
   * Fixed title: HDAC6 PPIs --> HDAC6 interactions in the central nervous system
 * 1 no longer active and perhaps not fit for WikiPathways: [WP5361](https://classic.wikipathways.org/index.php/Pathway:WP5361)
   * contains zero genes and metabolites; not a molecular pathway

### New pathways

 * [WP5425](https://classic.wikipathways.org/index.php/Pathway:WP5425) - Metabolism of T-2 Toxin and HT-2 Toxin (Triticum aestivum)
 * [WP5426](https://classic.wikipathways.org/index.php/Pathway:WP5426) - HDAC6 interactions in the central nervous system (Homo sapiens)

### Deleted Pathways

 * None

### Additional notes

 * Added deletion tag to  [WP5361](https://classic.wikipathways.org/index.php/Pathway:WP5361): contains zero genes and metabolites; not a molecular pathway

### Update Approved pathway tag

- [X] All Approved pathways tags were updated!

**Comment by danidi** [comment]
[WP5361](https://classic.wikipathways.org/index.php/Pathway:WP5361) is an adverse outcome pathway, maybe @marvinm2 wants to comment on if it can be deleted or is still actively being worked on.

**Comment by Chris-Evelo** [comment]
That is correct. And we should discuss this I think. 

The concept of adverse outcome pathway (as introduced by OECD and gathered on AOPwiki) is that it consists of a molecular initiating event, a series of key events, and the adverse outcome. Our idea for molecular AOPs, described in Marvin's thesis, is that each of these events actually can be described as a molecular pathway, and often already is. That makes it interesting to use "meta-pathways" that describe the AOP itself, mention all the key events, and link to the actual molecular pathways themselves. That not only gives us a handy overview and allows some new kinds of analysis, but also ensures that we do not have to add pathway links to the individual pathways that describe key events. That is a good thing since that kind of links might be confusing as these events play many more roles in biology besides being part of an AOP. 

So, yes, these meta pathways do typically not have genes or metabolites (sometimes the molecular initiating event has just a single target, and then it makes sense to mention that) but they do have active links to other pathways and are useful in specific types of analysis. They are good to have for that purpose, but we might want to label them differently and put them in a specific collection because they would not be useful for direct molecular analysis. They are already linked to the AOP community.

**Reply by danidi** [reply]
This one is not yet linked to the AOP community, it is also not yet approved. It has the tag that it needs work (missing literature references).

**Reply by Chris-Evelo** [reply]
Thanks @danidi, I did not realize that indeed.  But the thing is that that will likely not remove the "not a molecular pathway" reason @AlexanderPico gave to delete this pathway which would be an argument for other AOP meta pathways too. But maybe that was already solved and these do not appear in the normal analysis set?

**Reply by danidi** [reply]
Indeed, that is still an argument. Just wanted to mention it, because if the pathway is kept, the tag for the community should still be added. For me, at the moment, the main question is if somebody is still actively working on this, and will finish the pathway.

**Reply by khanspers** [reply]
Here are all the AOP pathways that are currently on the new site (and therefor part of the "Analysis" set): https://www.wikipathways.org/communities/aop.html

If pathways are missing and should be added, use the pencil icon on that page to add pathways to the community.
